src/Agent.Worker/TaskManager.cs

```diff
@@ -583,17 +598,22 @@ public string WorkingDirectory
 
     public sealed class NodeHandlerData : BaseNodeHandlerData
     {
-        public override int Priority => 2;
+        public override int Priority => 3;
     }
 
     public sealed class Node10HandlerData : BaseNodeHandlerData
+    {
+        public override int Priority => 2;
+    }
+
+    public sealed class Node14HandlerData : BaseNodeHandlerData
```

Comment:
I thought about abstracting the idea of a "node" handler so we have one "NodeHandlerData" that takes in a version. I decided against this because Node10 is already a separate handler and it makes more sense to keep this pattern consistent for task authors.